performance-tuning/99-smp-packet-steering-minim-defaults

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2023 Minim, Inc.
+
+. /etc/device_info
```

Comment:
This would be the 'openwrt' way to do it.  I guess one benefit of this way is that if you end up with more than one board for which you want to do the same thing, you can add it to the list of boards for a particular case without having to create a new function for it:
```
. /lib/functions.sh

uci delete minim.@smp_packet_steering[0]
uci add minim smp_packet_steering

case $(board_name) in
        motorola,r14)
              uci add_list minim.@smp_packet_steering[0].irqs="120:8"
              uci add_list minim.@smp_packet_steering[0].irqs="121:4"
        ;;
esac

uci commit
```